src/config/get-config.ts

```diff
@@ -81,6 +81,15 @@ export interface Config {
    * is set to `false`, then the feature will be available in all channels
    */
   AUTO_LINK_CHANNEL: string | false;
+  /**
+   * The role to assign to users who wish to be notified of new live streams.
+   */
+  STREAM_NOTIF_ROLE: string;
+  /**
+   * The message ID for the livestream role assignment - users react to this
+   * message to be assigned the role.
+   */
+  STREAM_MSG_ID: string;
```

Comment:
How does one get this id again? Maybe we should update the comment rough instructions on how to get it.

Reply:
That'll be something I slap in the .env once I post the message - after that we shouldn't have to touch it, but I can add instructions.

Reply:
Updated the comment!